interactions/client/bot.py

```diff
@@ -359,7 +359,7 @@ def decorator(coro: Coroutine):
             return coro
 
         if coro is not MISSING:
-            self._websocket._dispatch.register(coro, coro.__name__)
+            self._websocket._dispatch.register(coro, name if name is not MISSING else coro.__name__)
             # it is not possible to provide a name here since it is only called when you use `@event`
```

Comment:
@EdVraz I guess you were wrong since the commands literally use `return self.event(coro, name=f"command_{name}")`
Sorry, did not notice this before

Reply:
@dacid44 please remove these comments

Reply:
Removed the comments